packages/react-query/package.json

```diff
@@ -22,6 +22,7 @@
     ".": {
       "import": {
         "types": "./build/modern/index.d.ts",
+        "react-server": "./build/modern/server.js",
         "default": "./build/modern/index.js"
       },
       "require": {
```

Comment:
Do you know for sure that the server will always use ESM? We have seen some bundlers still use `require` here, so that might be the reason why it's not picking up the import.

Reply:
does that mean we have to add the `react-server` entry to `require`, too ?

Reply:
That might be the case.